pyresample/_formatting_html.py

```diff
@@ -0,0 +1,356 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2010-2020 Pyresample developers
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Functions for html representation of area definition."""
+
+import uuid
+from functools import lru_cache
+from html import escape
+from importlib.resources import read_binary
+
+import numpy as np
+
+import pyresample.geometry as geom
+
+try:
+    import cartopy
+    cart = True
+except ModuleNotFoundError:
+    cart = False
```

Comment:
This is a personal preference, but you could avoid the extra boolean variable (here and for the xarray case) and do `cartopy = None` in the except case.

Reply:
Do you want me to change it?

Reply:
Sure. It's just me being nitpicky.